File: tests/fixtures/language_adapter_symbols/sample.zig
// HAND-DERIVED: standard Zig syntax written from language knowledge, not from any adapter regex.
const std = @import("std");

pub const Item = struct {
    name: []const u8,
    quantity: u32,
};

pub fn addQuantity(a: u32, b: u32) u32 {
    return a + b;
}

File: tests/fixtures/language_adapter_symbols/Sample.swift
// HAND-DERIVED: standard Swift syntax written from language knowledge, not from any adapter regex.
import Foundation

struct Item {
    var name: String
    var quantity: Int
}

class Warehouse {
    private var items: [Item] = []

    func addItem(_ item: Item) {
        items.append(item)
    }
}

protocol Describable {
    func describe() -> String
}
